components/spi/kustomization.yaml

```diff
@@ -1,16 +1,16 @@
 resources:
-  - https://github.com/redhat-appstudio/service-provider-integration-operator/config/default?ref=ccc1d8a0b400aa3e47c10fa14040674252d3ae3d
-
+  - https://github.com/sparkoo/service-provider-integration-operator/config/default?ref=c36682a4fd7dbac2e7303637e0cc4c5ff958cfd4
```

Comment:
use redhat-appstudio org, the commit is there

Reply:
fixed